.eslintrc.json

```diff
@@ -4,6 +4,15 @@
     "Prism": false
   },
   "rules": {
-    "linebreak-style": ["error", "unix"]
-  }
+    "linebreak-style": ["error", "unix"],
+    "no-console": ["error", { "allow": ["warn", "error"] }]
```

Comment:
For this part, I think we need to fix [the config that's being extended](https://github.com/Brightspace/eslint-config-brightspace/blob/master/index.js).

Reply:
That is, I think `eslint:recommended` ruleset changed within es-lint, and we didn't realize.

Reply:
Agreed, and as long as we do a minor release because it's still `<0` it'll be treated like a major one.